org-roam.el

```diff
@@ -364,16 +364,16 @@ If PREFIX, downcase the title before insertion."
           (when region ;; Remove previously selected text.
             (delete-region (car region) (cdr region)))
           (insert (org-roam--format-link target-file-path link-description)))
-      (if org-roam-capture--in-process
-          (user-error "Nested Org-roam capture processes not supported")
-        (let ((org-roam-capture--info (list (cons 'title title)
-                                            (cons 'slug (org-roam--title-to-slug title))))
-              (org-roam-capture--context 'title))
-          (add-hook 'org-capture-after-finalize-hook #'org-roam-capture--insert-link-h)
-          (setq org-roam-capture-additional-template-props (list :region region
-                                                                 :link-description link-description
-                                                                 :capture-fn 'org-roam-insert))
-          (org-roam--capture))))))
+      (when (org-roam-capture--in-process-p)
+	(user-error "Nested Org-roam capture processes not supported"))
+      (let ((org-roam-capture--info (list (cons 'title title)
+					  (cons 'slug (org-roam--title-to-slug title))))
```

Comment:
```suggestion
      (let ((org-roam-capture--info `((title . ,title) (slug . ,(org-roam--title-to-slug title))))
```

Reply:
Thanks for your Feedback. I prefer to follow git best practices and use single-purpose commits. 
I myself often got sidetracked to code cleanups an introduced non-related bugs :roll_eyes: